package.json

```diff
@@ -39,7 +39,7 @@
     "copy-to-clipboard": "3.2.0",
     "deep-diff": "1.0.2",
     "emotion-theming": "^10.0.10",
-    "giantswarm": "git+https://github.com/giantswarm/giantswarm-js-client.git",
+    "giantswarm": "git+https://github.com/giantswarm/giantswarm-js-client.git#app-secrets",
```

Comment:
TODO: Merge https://github.com/giantswarm/giantswarm-js-client/pull/78 and change this back to master